metricbeat/module/kubernetes/util/metrics_cache.go

```diff
@@ -68,6 +59,41 @@ func (c *PerfMetricsCache) Stop() {
 	c.ContainerCoresLimit.Stop()
 }
 
+func (c *PerfMetricsCache) GetTimeout() time.Duration {
```

Comment:
Could you add a docstring here to document this method and the logic encapsulated?